pkg/apiserver/apiserver.go

```diff
@@ -2,26 +2,30 @@ package apiserver
 
 import (
 	"github.com/openshift/oauth-apiserver/pkg/serverscheme"
+	"github.com/openshift/oauth-apiserver/pkg/tokenvalidation"
 	genericapiserver "k8s.io/apiserver/pkg/server"
 	restclient "k8s.io/client-go/rest"
 
 	openshiftcontrolplanev1 "github.com/openshift/api/openshiftcontrolplane/v1"
+
 	oauthapiserver "github.com/openshift/oauth-apiserver/pkg/oauth/apiserver"
 	userapiserver "github.com/openshift/oauth-apiserver/pkg/user/apiserver"
 	"github.com/openshift/oauth-apiserver/pkg/version"
 )
 
 type Config struct {
-	GenericConfig *genericapiserver.RecommendedConfig
+	GenericConfig         *genericapiserver.RecommendedConfig
+	TokenValidationConfig *tokenvalidation.TokenValidationConfig
 }
 
 type OAuthAPIServer struct {
 	GenericAPIServer *genericapiserver.GenericAPIServer
 }
 
 type completedConfig struct {
-	GenericConfig genericapiserver.CompletedConfig
-	ClientConfig  *restclient.Config
+	GenericConfig         genericapiserver.CompletedConfig
+	ClientConfig          *restclient.Config
+	TokenValidationConfig tokenvalidation.TokenValidationConfig
```

Comment:
elsewhere we have the style of `ExtraConfig`, subsuming everything in addition. It makes it easier to just copy everything in the completion func.